core/bessctl.cc

```diff
@@ -1241,10 +1241,10 @@ class BESSControlImpl final : public BESSControl::Service {
 
     if (is_any_worker_running()) {
       propagate_active_worker();
-      if (m1->num_active_workers() || m2->num_active_workers() ) {
-	WorkerPauser wp; // Only pause when absolutely required
-	ret = m1->ConnectModules(ogate, m2, igate);
-	goto done;
+      if (m1->num_active_workers() || m2->num_active_workers()) {
+        WorkerPauser wp;  // Only pause when absolutely required
```

Comment:
FYI I think this reformat might be the same as one in one of @melvinw PRs, hopefully they match exactly and Git will be able to handle it easily 😺